utils/kata-manager.sh

```diff
@@ -254,14 +254,17 @@ pre_checks()
 {
 	info "Running pre-checks"
 
+	local only_kata="${1:-}"
+	[ -z "$only_kata" ] && die "no only_kata value"
+
 	command -v "${kata_shim_v2}" &>/dev/null \
 		&& die "Please remove existing $kata_project installation"
 
 	local ret
 
 	{ containerd_installed; ret=$?; } || true
 
-	[ "$ret" -eq 0 ] && die "$containerd_project already installed"
+	[ "$ret" -eq 0 ] && [only_kata = "false" ] && die "$containerd_project already installed"
```

Comment:
It would be better to check for `only_kata` at the start of the function.
If it's true, we simply return 0, if not we proceed with function logic.  By doing this we avoid actually checking something that will never be used in case `only_kata` is true.

Reply:
Sounds good, still keeping the kata-is-already-installed check, though. Fixed.